File: 1ºAno/Lista2/Python/Sala/files/ex1.py
notas = [float(input("Entre com as notas:\nN1 = ")), float(input("N2 = ")), float(input("N3 = "))]
print ("\nAprovado :)") if (sum(notas) / 3 >= 6) else print ("\nReprovado :(")
print ("Média = %.2f" % (sum(notas) / 3))

File: 1ºAno/Lista2/C/Sala/files/ex1.c
#include <stdio.h>

int main() {

    float notas[3];
    printf ("Entre com as notas:\n");
    for (int i = 1; i <= 3; i++) {
        printf ("N%d: ", i);
        scanf ("%f", &notas[i - 1]);
    }
    printf ("Média = %.2f\n", (notas[0] + notas[1] + notas[2]) / 3);
    if ((notas[0] + notas[1] + notas[2]) / 3 >= 6) {
        printf ("Aluno aprovado :)\n");
    }
    else {
        printf ("Aluno reprovado :(\n");
    }

    return 0;
}